src/main/scala/wdl4s/Call.scala

```diff
@@ -180,7 +180,7 @@ sealed abstract class Call(val alias: Option[String],
         parentLookup <- Try(parent.lookupFunction(inputs, wdlFunctions, outputResolver, shards, relativeTo)(name))
       } yield parentLookup
 
-      val resolutions = Seq(inputMappingsLookup, declarationLookup, declarationExprLookup, taskParentResolution)
+      val resolutions = Seq(inputMappingsLookup, declarationExprLookup, declarationLookup, taskParentResolution)
```

Comment:
I'm not sure why this wasn't done with lazy streams to avoid needlessly running lower priority resolvers, but I chose to keep the changes minimal for this PR.